<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Alex PC Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_INFO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>系统版本号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>81</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU型号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SystemVersion">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_CPUType">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>79</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>启动时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>运行时长：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>主机名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_hostname">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>AlexFanLinux</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_boottime">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2020.3.1 18:30:10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_runningtime">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>00：00</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_process">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>391</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>315</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU使用率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内存使用率：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shutdown">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>420</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关   机</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CurrentTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_RAM">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="cpu_rate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>315</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>pushButton_shutdown</zorder>
   <zorder>label_CurrentTime</zorder>
   <zorder>progressBar_RAM</zorder>
   <zorder>tabWidget_INFO</zorder>
   <zorder>cpu_rate</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
